.buildkite/pipeline.yml

```diff
@@ -0,0 +1,106 @@
+env:
+  BUILDKITE_PLUGINS_ALWAYS_CLONE_FRESH: 1
```

Comment:
Is this still needed, or was this only added during debugging but could be removed now?

Reply:
Oh I see I guess this is related to https://github.com/Automattic/simplenote-electron/pull/3204/files#r1637752108 and will be removed once you land that to `a8c-ci-toolkit`'s trunk` 👍